opm/simulators/wells/MultisegmentWellAssemble.cpp

```diff
@@ -116,7 +116,7 @@ assembleControlEq(const WellState& well_state,
         return rates;
     };
 
-    if (well_.wellIsStopped()) {
+    if (well_.wellUnderZeroRateControl(summaryState, well_state)) {
```

Comment:
One big question here is that, with the current form, we include the `wellIsStopped` under the category `wellUnderZeroRateControl`. 

If you think that it is more readable by separating `wellIsStopped` from `wellUnderZeroRateControl`, then for multiple places we do 
`this->wellIsStopped() || this->wellUnderZeroRateControl()` , 
please suggest.

Reply:
> Maybe in the future when we see they can be for sure be one, let us deal it then.

We might also just make a helper function that returns
```C++
this->wellIsStopped() || this->wellUnderZeroRateControl()
```
and use that.  It's not clear to me what to name that function though.  `IsBehaviourllyStopped()`, maybe?

Reply:
> maybe `stopppedOrZeroRateControl`?

I like it.  Maybe even `stoppedOrZeroRateTarget()`?